tox.ini

```diff
@@ -16,5 +16,5 @@ commands =
   yamllint -s .
 
 [flake8]
-select = F401,F402,F821,F823,F841,F811
+select = F401,F402,F821,F823,F841,F811,E265,E266,F541,W605,E722,F822,F523,W291,F405
```

Comment:
Just to note what's going on here:

- E265: Block comment should start with '# '
- E266: Too many leading '#' for block comment
- F541: f-string without any placeholders
- W605: Invalid escape sequence 'x' (backslash escapes)
- E722: Do not use bare except, specify exception instead
- F822: undefined name name in __all__
- F523: .format(...) unused positional arguments
- W291: Trailing whitespace
- F405: Name may be undefined, or defined from star imports: module

I wholeheartedly endorse.

Reply:
... and then I see after the fact that you had already put this info in the main comment on this PR.  :man_facepalming:

Reply:
It sure would be nice to have these as a comment in this same file. But then I Google and find an RFE on that closed as wontfix